### PR TITLE
For /languages/:language, allow search for Github issues based on labels

### LINKED ISSUE #246: Allow overriding languages for issues on GitHub repos by the usage of labels
If I open https://codetribute.mozilla.org/languages/javascript, I see issues from the `mozilla/firefox-code-coverage-frontend` project, but I don't see issues from the `marco-c/code-coverage-reports` (even though issues from `mozilla/firefox-code-coverage-frontend` don't have a language label, issues from `marco-c/code-coverage-reports` do).

How can the language "skill" be specified for projects (or better, issues, since in several of my projects there are issues where different skills are needed)?

### PR DESCRIPTION
**Pull request by fiennyangeln**
Use `lang=language` to standardize it with bugzilla's

Fix #246 

Note: I try to use `lang=${language}` instead of the language directly as the label to make the search more specific.

### CODE REVIEW

**Comment by fiennyangeln** [comment]
Agree! Let me include it here 👍

File: README.md [hunk 1]
```diff
@@ -145,6 +145,8 @@ repositories:
  - mozilla/telemetry-dashboard: ['mentored', 'good first issue']
 ```
 
+For the `/languages/:language` route, we get issues tagged with labels in README from repositories whose primary language is the language or those labelled `lang:[language]`.
+
```
Comment (djmitche) [inline README.md]: `:` -> `=`

**Comment by fiennyangeln** [comment]
I modified the readme. 😃

File: README.md [hunk 2]
```diff
@@ -145,6 +145,8 @@ repositories:
  - mozilla/telemetry-dashboard: ['mentored', 'good first issue']
 ```
 
+For the `/languages/:language` route, we get issues tagged with labels in README from repositories whose primary language is the language or those labelled `lang=[language]`.
```
Comment (helfi92) [inline README.md]: > we get issues tagged with labels in README

I'm not sure I understand the link with the `README`. Could you help me understand, please?

**Comment by marco-c** [comment]
Would it be possible to allow each project to define their own labeling scheme, like Bugsahoy did?

File: README.md (hunk 2 above)
Comment (fiennyangeln) [reply README.md]: 🙈aah my bad! I mean markdown file 🙈

**Comment by helfi92** [comment]
From what I can see, users on Github tend to use only the language as the main label. A quick search can demonstrate that:

[`lang=javascript`](https://github.com/search?q=label%3A%22lang%3Djavascript%22): 0 issues
[`lang:javascript`](https://github.com/search?q=label%3A%22lang%3Ajavascript%22): 455 issues
[`javascript`](https://github.com/search?q=label%3A%22javascript%22): 23k issues

[`lang=python`](https://github.com/search?q=label%3A%22lang%3Dpython%22): 0 issues
[`lang:python`](https://github.com/search?q=label%3A%22lang%3Apython%22): 128 issues
[`python`](https://github.com/search?q=label%3A%22python%22): 14k issues

That said, it makes sense to remove the `lang=` prefix.

**Comment by djmitche** [comment]
I think 35 queries to github for each hit is going to lead to us exceeding our github API request limit when we need it most!  Let's err on the side of reducing total number of queries.

**Comment by helfi92** [comment]
@fiennyangeln Feel free to remove the `lang=${language}` requirement and make it only `${language}`.

**Comment by fiennyangeln** [comment]
@helfi92 done! 
@marco-c hope this fix helps you! I will look into batching multiple queries once the graphql server supports it

File: src/views/Languages/index.jsx [hunk 3]
```diff
@@ -202,7 +202,7 @@ export default class Languages extends Component {
 
     this.setState({ githubLoading: true });
 
-    tagsMapping[`lang=${language}`] = Object.keys(repos);
+    tagsMapping[`${language}`] = Object.keys(repos);
```
Comment (helfi92) [inline src/views/Languages/index.jsx]: Any reason this is not `tagsMapping[language]`?

File: README.md [hunk 4]
```diff
@@ -146,7 +146,7 @@ repositories:
 ```
 
 For the `/languages/:language` route, we get issues from repositories whose primary language is the language selected & tagged with labels specified in yaml files
-or any issues across Github repositories labelled `lang=[language]`.
+or any issues across Github repositories labelled the language selected.
```
Comment (helfi92) [inline README.md]: ``` s/labelled the language selected/with the language lowercased as the label (e.g., `javascript`)/ ```

**Pull request merged**